composer.json

```diff
@@ -7,17 +7,29 @@
     "php": "^5.2|^7"
   },
   "require-dev": {
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-    "phpcompatibility/php-compatibility": "^9.1",
-    "squizlabs/php_codesniffer": "^3.4",
-    "wp-coding-standards/wpcs": "^1.2"
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.6.2",
+    "matthiasmullie/minify": "^1.3.63",
+    "phpcompatibility/php-compatibility": "^9.3.5",
+    "squizlabs/php_codesniffer": "^3.5.5",
+    "wp-coding-standards/wpcs": "^2.3.0"
   },
   "scripts": {
+    "post-install-cmd": [
+      "@build"
+    ],
+    "post-update-cmd": [
+      "@build"
+    ],
+    "build": [
+      "minifyjs assets/js/notice.js > assets/js/notice.min.js"
+    ],
     "cs": [
-      "phpcs --standard=phpcs.xml -s"
+      "phpcs --standard=phpcs.xml -s",
+      "npx eslint assets/js/notice.js"
     ],
     "csfix": [
-      "phpcbf --standard=phpcs.xml"
+      "phpcbf --standard=phpcs.xml",
+      "npx eslint assets/js/notice.js --fix"
```

Comment:
I'd personally have split the checks for PHP and JS (considering Composer-only environments) with wrapper steps, although it's only a single line here.

Reply:
That's an argument, for changes that don't affect the JavaScript I should be valid to ignore the JavaScript code style check in the development environment). So I added separate commands for the checks as well.